projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -447,12 +243,7 @@
   }
 
   .clr-filter-highlight b {
-    font-weight: variables.$clr-font-weight-semibold;
-    @include mixins.css-var(
-      color,
-      clr-combobox-filter-highlight,
-      combobox-variables.$clr-combobox-filter-highlight,
-      variables.$clr-use-custom-properties
-    );
+    font-weight: tokens.$cds-global-typography-font-weight-semibold;
```

Comment:
Where this `font-weight` override is used and why is needed?

Reply:
On the filter highlight directive, which exists but is not used, due to legacy security concerns.
Already talked with Delchina that we should have it discussed with PM if we want to reenable this feature.
The idea is to highlight (bold in this case) substrings of the suggestions list items that match the input text.